core/src/main/java/com/facebook/testing/screenshot/internal/ScreenshotDirectories.java

```diff
@@ -30,7 +31,8 @@ public File get(String type) {
   private File getSdcardDir(String type) {
 
     String parent = String.format(
-      "/sdcard/screenshots/%s/",
```

Comment:
unfortunately I'm reverting this specific change. We do care about the specific directory since it's referred from the python side code and this breaks the unit tests right now.